### PR TITLE
Feature/multiple upvotes

### PR DESCRIPTION
**Pull request by knharman**
The association with the votes was incorrect - it needed to be tied to the session user id, not the recipe creator id.

### CODE REVIEW

File: controllers/api/vote-routes.js [hunk 1]
```diff
@@ -7,49 +7,19 @@ const error500 = err => {
   res.status(500).json(err);
 };
 
-// insomnia test PUT /api/posts/upvote
-router.put('/vote', (req, res) => {
```
Comment (knharman) [inline controllers/api/vote-routes.js:11]: this end point was not getting used

File: controllers/api/vote-routes.js [hunk 2]
```diff
@@ -7,49 +7,19 @@ const error500 = err => {
...
-          recipe_id: req.params.id
+          recipe_id: req.params.id, 
+          user_id: req.session.user_id
         }
       })
-      if(count == 0) {
+      if (count == 0) {
         await Vote.create({
-          user_id: recipe.user.id, 
+          user_id: req.session.user_id,
```
Comment (knharman) [inline controllers/api/vote-routes.js:22]: needed to use the user id from the session for currently logged in user to add votes, not the recipe creator id

File: controllers/homepage-routes.js [hunk 3]
```diff
@@ -32,6 +32,15 @@ router.get("/", (req, res) => {
...
       const recipes = dbRecipeData.map((recipe) => recipe.get({ plain: true }));
+      recipes.sort((recipeOne, recipeTwo) => {
+        if (recipeOne.vote_count < recipeTwo.vote_count) {
+          return 1
+        }
+        if (recipeOne.vote_count > recipeTwo.vote_count) {
+          return -1
+        }
+        return 0
+      })
```
Comment (knharman) [inline controllers/homepage-routes.js:43]: this change was in main, but not develop so I am merging it in to keep develop up to date

**Review by DrDano — APPROVED** [review]
Looks good! Thanks for doing this Kellie!

**Pull request merged**